main_logic/tts_client.py

```
@@ -1098,6 +1098,292 @@ def gemini_tts_worker(request_queue, response_queue, audio_api_key, voice_id):
+            resampler = soxr.ResampleStream(current_rate, 48000, 1, dtype='float32')
+            ws = await websockets.connect(ws_url, additional_headers=headers, ping_interval=None)
+            receive_task = asyncio.create_task(receive_loop(ws))
+            return ws
+
+        async def flush_pending_payloads():
+            nonlocal ws, pending_payloads
+            if not pending_payloads:
+                return
+            try:
+                await create_connection()
+            except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)